core/commit.go

```diff
@@ -116,6 +116,10 @@ func makeCommitmentCollector(countCommitment commitmentCounter, retireSeq reques
 		}
 
 		pendingReq.Remove(request.Msg.ClientId)
+
+		// FIXME: Only stop the request timer if the request
+		// identifier it was started for matches the one of
+		// the request accepted for execution.
```

Comment:
I think that when the newer request comes, request timer for the client is restarted by request applier, so the newer request seems not to be prevented by request timer started by the older request? But maybe situation is more complicated by view change...